src/main/java/org/amahi/anywhere/fragment/UploadSettingsFragment.java

```diff
@@ -111,10 +112,27 @@ private void setUpSettings() {
 
     private void setUpSettingsContent() {
         addPreferencesFromResource(R.xml.upload_settings);
+        setUploadPathSummary(getUploadPath());
+    }
+
+    private void setUploadPathSummary(String path){
+        if(!path.isEmpty()){
+            if(path.length()>25){
+                path = path.substring(0,21);
+                path += "...";
+            }
+            getPathPreference().setSummary(path);
+        }
     }
 
     private void setUpSettingsTitle() {
         getAutoUploadSwitchPreference().setTitle(getAutoUploadTitle(isUploadEnabled()));
+
+        if (isConnectionLocal()) {
+            getHdaPreference().setTitle(getString(R.string.preference_title_upload_server));
+        } else {
+            getHdaPreference().setTitle(getString(R.string.preference_title_upload_hda));
+        }
```

Comment:
I don't think we need two different titles for the same field. `HDA Server` and `Server`.
Make it just `Server` as pointed out in 2nd point of the [comment](https://github.com/amahi/android/pull/328#issuecomment-372926236) by cpg.

Reply:
Sorry, I got confused by the 2nd point of the comment. I thought It means to change the title to `Server` when connecting to local and otherwise `HDA Server`.
I will make it `Server`.